ci/build_cuda_parallel_python.sh

```diff
@@ -25,8 +26,7 @@ docker run --rm -i \
     source /etc/profile.d/enable_devtools.sh && \
     which python && \
     python -m pip wheel --no-deps . && \
-    python -m pip install patchelf auditwheel && \
+    python -m pip install auditwheel && \
     python --version && \
-    python -m auditwheel repair cuda_parallel-*.whl --exclude libcuda.so.1 && \
-    mv cuda_cccl-*.whl /workspace/cccl/wheelhouse && \
-    mv wheelhouse/cuda_parallel-*.whl /workspace/cccl/wheelhouse/'
+    python -m auditwheel show cuda_parallel-*.whl && \
```

Comment:
Here you would keep `libcuda.so.1` and add in additional `--exclude libnvrtc.so.*` etc.

Reply:
Did you see that I changed from `repair` → `show`? The latter will have a return code other than `0` if the audit step is failing. I'm thinking we don't have anything to "repair"?

Reply:
We do want to still run repair in case the [wheel triplet](https://packaging.python.org/en/latest/specifications/binary-distribution-format/#file-name-convention) is not set properly.

Reply:
Got it, thanks!

1. commit aaf8fcadd7dae022ad40a91a51802af6095086a7
2. commit ba3b2100b65cf6b9b239d3c6c1451b49996ba65c
3. commit daea174b08018e62057cdf32a0af42a3ba8fa985